features/netsocket/UDPSocket.h

```diff
@@ -112,6 +112,8 @@ class UDPSocket : public Socket {
     nsapi_size_or_error_t recvfrom(SocketAddress *address,
             void *data, nsapi_size_t size);
 
+    nsapi_error_t set_broadcast(bool broadcast);
```

Comment:
Would you be able to add the appropriate doxygen documentation to this function. You can use the neighboring function's documentation as a template
https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/Socket.h#L109-L120